documents/fsd/chapter/func.tex

```diff
@@ -70,7 +70,7 @@ \section{User interface specification}
     \fspec[300]{Recording indicator}{\Gls{user}}{\applaunched{} \sessioninactive{} \configvalid}{The \gls{user} clicks on the button labeled "Start Recording". The user-interface-managing component will start displaying a yellow border along the screen edges.}{A yellow border is being drawn along the screen edges as long as the recording is active.}
     \fspec{Error dialog}{\Gls{user}}{\configinvalid}{The \gls{user} starts the application. The configuration-managing component will detect syntactic errors in the configuration file and the user-interface-managing component will display an error-message.}{An error dialog is displayed, informing that the current configuration is invalid. When the \gls{user} clicks the button labeled "OK" displayed in this dialog, the application will terminate.}
     \fspec{Save dialog}{\Gls{user}}{\sessionactive}{The \gls{user} stops the recording by pressing the button labeled "Stop recording" or the button labeled "Quit". The user-interface-managing component opens a new save dialog.}{A dialog window is displayed to the \gls{user}, providing the options "Save recording" and "Discard recording".}
-    \fspec{Event-Data extraction}{\Gls{scientist}}{\sessionsavail}{A \gls{scientist} starts the supplied command-line-tool and supplies the paths of one or multiple recordings as well as an output path as parameters. The tool will start extracting the stored event-data from the recording.}{The event-data has been extracted from the specified recordings and stored in JSON-format (JavaScript Object Notation) at the location specified as output path. The recordings specified as input files have not been altered.}
+    \addtocounter{counterFS}{10} %% removed FS330
     \fspec{Tray-Icon}{\Gls{user}}{\configvalid}{The \gls{user} starts the application. The user-interface-managing component adds a new icon to the tray, indicating that the software has been launched successfully.}{An additional icon is displayed in the system tray.}
```

Comment:
```suggestion
    \fspec[340]{Tray-Icon}{\Gls{user}}{\configvalid}{The \gls{user} starts the application. The user-interface-managing component adds a new icon to the tray, indicating that the software has been launched successfully.}{An additional icon is displayed in the system tray.}
```

Instead of adding to the counter, just set the FS-number directly; this will only be useful if we add anything after FS320, but I would still prefer this approach.